Java-Control-Statements.md

```diff
@@ -0,0 +1,216 @@
+#Java Control Statements :
+
+**In** any programming languege, the flow of program is neccessary for executing the program and achieving desired results.
+`Control statements` manage the flow of execution of the program and decide the branching of logical flow. Java's program 
+control statements can be put into following categories: `selection` , `iteration` and `jump statement` . `Selection statement`
+allows a program to choose different path of executions depending on an expression. `Iteration` allows a program to repeat 
+one or more statements. `Jump statement` is used to transfer control to another part of the program. As `iteration` and `jump
+statements` are already covered in `Java-loops`, here we would discuss `Selection statements`.
+
+##Java's selection statement :
+
+`Selection statements` allow a program to choose different executions based on the outcome of an `expression` or `state of a variable
+`.
+Java has two selection statements : `if-else` and `switch`.
+
+###if-else
+
+`if-else` is a conditional branch statement that helps us to choose between two available choices. The general syntax of `if-else` is
+shown below :
+
+```
+if(condition)
+statement1;
+else
+statement2;
+```
+
+In the above code snippet, the `condition` is first resolved. If it turns out to be true, then the statements that are inside the if 
+condition are executed(`statement1 in this case`). If the `condition` turns out to be false, then the statements inside else are 
+executed(`statement2, in this case`). 
+
+**Example :**
+
+Here is an example usage which states whether a number is an `even number or an odd number`. Even numbers are divisible by two
+and hence leave a remainder of zero. The remainder on dividing the number by two can be obtained using the `modulo (%) mathematical
+operator`. 
+
+```
+public class Test {
+   public static void main(String args[])
+      {
+   int num=5;
+   
+if ( num%2==0){ 
+    System.out.println("Number is even");
+}
+else{
+    System.out.println("Number is odd");
+       }
+   }
+}
+```
```

Comment:
Create a [REPL](https:/repl.it) code snippet and link it here. It would be easier to test out the code at one click.

@koustuvsinha maybe you can show him how to do this?